### DISCUSSION TITLE
Feature suggestion: RTSP or ONVIF support

### DISCUSSION THREAD

**Discussion by Emstar** [easytarget/esp32-cam-webserver#298] [Ideas]
If the camera server supported the RTSP or ONVIF protocols, it would be possible to integrate these cheap cams with other devices such as IP camera network video recorders. Ingesting the feed into an NVR would be _really_ handy. For one thing it would let you work around the single-user nature of the current stream. 

I did find an RTSP server project, and it does kind of work, but it generates loads of udp errors and drops a lot of frames. Not sure what the problem is, but maybe this code would be a good starting point? 

https://github.com/circuitrocks/ESP32-RTSP

**Comment by easytarget** [comment]
Good suggestions; I know that there are other esp32-cam projects implementing these protocols these days, but I dont have a list of them.. ;-)

Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
